scratch/dean/ray_window_timing-20230124.csv

```diff
@@ -0,0 +1,17 @@
+,num_blocks,blocks_per_window,time
```

Comment:
Let's put these CSV files in the new scratch repo https://github.com/HazyResearch/meerkat-dev